PresentScreenings/Entities/Screening.cs

```diff
@@ -206,7 +209,9 @@ public void ToggleFilmFanAttendance(string filmFan)
 
         public bool HasPlannableStatus()
         {
-            return Status == ScreeningInfo.ScreeningStatus.Free && !SoldOut;
+            bool free = Status == ScreeningInfo.ScreeningStatus.Free;
+            bool attendedByFriend = Status == ScreeningInfo.ScreeningStatus.AttendedByFriend;
+            return (free || attendedByFriend && TimesIAttendFilm == 0 && !HasNoTravelTime) && !SoldOut;
```

Comment:
Can't this be simplified?
Since the first term is:
`free == (!attendedByFriend && TimesOAttendFilm == 0 && !HasNoTravelTime)`,
and the second term is:
`attendedByFriend && TimesIAttendFilm == 0 && !HasNoTravelTime`
we can use:
`!A && B || A && B` is `(!A || A) && B` is `B`
 so that the returned result can be reduced to:
`return TimesIAttendFilm == 0 && !HasNoTravelTime && !SoldOut`

Reply:
Done.